support/build/extensions/no-debug-non-zts-20121212/sundown

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+dep_name=$(basename $BASH_SOURCE)
+
+source $(dirname $BASH_SOURCE)/../pecl
```

Comment:
Changing the line to ``source $(dirname $BASH_SOURCE)/../../../../buildpack/support/build/extensions/pecl`` does work. It's quite an ugly path, but better then maintaining a copy of ``pecl``. I will change the PR.